feature-libs/cart/saved-cart/components/saved-cart-form/saved-cart-form.component.ts

```diff
@@ -0,0 +1,57 @@
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+import { Cart } from '@spartacus/core';
+import { ICON_TYPE, LaunchDialogService } from '@spartacus/storefront';
+import { Observable } from 'rxjs';
+
+@Component({
+  selector: 'cx-saved-cart-form',
+  templateUrl: './saved-cart-form.component.html',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class SavedCartFormComponent implements OnInit {
+  form: FormGroup;
+  iconTypes = ICON_TYPE;
+  cart$: Observable<Cart>;
+
+  descriptionMaxLength: number = 500;
+  nameMaxLength: number = 50;
+
+  ngOnInit() {
+    this.cart$ = this.launchDialogService.data$;
+    this.build();
+  }
+
+  constructor(protected launchDialogService: LaunchDialogService) {}
+
+  get descriptionsCharacterLeft(): number {
+    return (
+      this.descriptionMaxLength -
+      (this.form.get('description')?.value?.length || 0)
+    );
+  }
+
+  saveCart(): void {
+    // TO DO
```

Comment:
To do later or leftover?

Reply:
This PR is only for UI part, this method is ready for state and facade PR.